### ISSUE TITLE
HTTP/2 support

### ISSUE THREAD

**Issue by sayoun** [pyrates/roll#61]
Hello,

do you plan to add HTTP/2 support to Roll in the future ?

**Comment by yohanboniface** [comment]
Yep, that's somewhere in the road map, see #54 that opened the path to this.

PRs are welcome if you feel so :)
